Dockerfile

```diff
@@ -0,0 +1,109 @@
+#{{{ demo_lti1p3
+FROM php:7.4-fpm-alpine as demo_lti1p3-base
+ENV MAKEFLAGS "-j8"
+
+RUN set -eux; \
+        cd /; \
+        apk add --no-cache --virtual .build-deps \
```

Comment:
if you check https://github.com/oat-sa/demo-lti1p3/blob/master/docker/phpfpm/Dockerfile#L9-L16, this is the only required extensions.

imo we should remove all the unused ones for faster and ligther builds, what doi you think ?

Reply:
Yes, you are right, I remove all unnecessary extensions, please check it